build/test-app-runtime.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env node */
+const path = require('path');
+const {spawn} = require('child_process');
+
+module.exports.TestAppRuntime = function({dir = '.', watch = false, match}) {
+  const state = {proc: null};
+
+  this.run = () => {
+    this.stop();
+
+    let command = require.resolve('jest-cli/bin/jest.js');
+    let args = [
+      // '--no-cache',
```

Comment:
I'm guessing this should be deleted?

Reply:
We can, though it's really useful for debugging.